doc/website/getting-started/examples/iceoptions.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Configure the history cache size and the starting behaviour
```

Comment:
Configure the history cache size and startup behavior

Reply:
Here it would have to be "Configuring ..."
I do not understand what starting behavior is supposed to tell me, it is kind of confusing to a reader I would say.

Reply:
Done. Hope you excuse me for favouring British English :innocent: 

I'm unsure whether to use present participle here or just presence. @ithier being a native speaker, volunteered to read the documentation mid next week :100: